sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/statusdisplay/RelativeCount.groovy

```diff
@@ -0,0 +1,76 @@
+package life.qbic.portal.sampletracking.components.projectoverview.statusdisplay
+
+import groovy.transform.EqualsAndHashCode
+
+/**
+ * <b>A relative count of something.</b>
+ *
+ * <p>This class represents a relative count. It has a value that is a part of a total.</p>
+ *
+ * @since 1.0.0
+ */
+@EqualsAndHashCode
+class RelativeCount implements Comparable<RelativeCount> {
+    final int value
+    final int total
+
+    RelativeCount(int value, int total) {
+        this.value = value
+        this.total = total
+    }
+
+    @Override
+    String toString() {
+        return "$value / $total"
+    }
+    /**
+     * Compares this object with the specified object for order.  Returns a
+     * negative integer, zero, or a positive integer as this object is less
+     * than, equal to, or greater than the specified object.
+     *
+     * <p>The implementor must ensure <tt>sgn(x.compareTo(y)) ==
+     * -sgn(y.compareTo(x))</tt> for all <tt>x</tt> and <tt>y</tt>.  (This
+     * implies that <tt>x.compareTo(y)</tt> must throw an exception iff
```

Comment:
Typo
```suggestion
     * implies that <tt>x.compareTo(y)</tt> must throw an exception if
```

Reply:
This is a Java faux-pas 😄  the javadoc was copied over.